apps/web-client/src/lib/db/orders/customers.ts

```diff
@@ -81,3 +81,23 @@ export const updateOrderLineQuantity = async (db: DB, bookId: number, quantity:
 	const params = [quantity, bookId];
 	await db.exec(sql, params);
 };
+
+export const markCustomerOrderAsReceived = async (db: DB, supplierOrderLines: SupplierOrderLine[]) => {
+	//query customer order lines with isbn and order them by placed
+	for (const line of supplierOrderLines) {
+		await db.execO<DBCustomerOrderLine>(
```

Comment:
When we're not interested in the result we should use `db.exec` instead of `db.execA` and `db.execO` in my opinion. Also, coderabbit has a good point.